index.js

```diff
@@ -653,32 +656,41 @@ class ClientServiceBase {
   }
 
   /**
-   * @param {ContractExecutionResult} result1
-   * @param {ContractExecutionResult} result2
-   * @return {Boolean}
-   * @throws {ClientError}
+   * @param {proto.rpc.ContractExecutionResponse} rsp1
+   * @param {proto.rpc.ContractExecutionResponse} rsp2
+   * @return {boolean}
    */
-  _validateResponses(result1, result2) {
+  _validateResponses(rsp1, rsp2) {
```

Comment:
I don't think the variable name `rsp` is allowed by the Google coding style.

Reply:
Ah! Thank you. Sorry about that.
Let me use `responseX` to replace them.

Reply:
I fixed this in https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/79/commits/60822a253ccbe88e54bec0429524d75262e7690e

Please take a look!